tests/haml/HamlLintBearTest.py

```diff
@@ -0,0 +1,116 @@
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coala_utils.ContextManagers import prepare_file
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from bears.haml.HamlLintBear import HamlLintBear
+
+
+test_file = """
+#content
+  .left.column
+    %h2 Welcome to our site!
+    %p= print_information
+  .right.column
+    = render partial: 'sidebar'
+"""
+
+config_file = """
+linters:
+    LineLength:
+        enabled: True
+        max: 80
+    TrailingWhitespace:
+        enabled: True
+"""
+
+with prepare_file(config_file,
+                  filename=None,
+                  force_linebreaks=True,
+                  create_tempfile=True) as (conf_lines, conf_file):
+    HamlLintBear1Test = verify_local_bear(
+                                         HamlLintBear,
+                                         valid_files=(test_file,),
+                                         invalid_files=(),
+                                         settings={
+                                            'hamllint_config': conf_file})
```

Comment:
Can you move this test inside the class ? We discourage the use of `verify_local_bear` so it would be good if you could use `check_results` or `check_validity`.